addons/binding/org.openhab.binding.tankerkoenig/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tankerkoenig"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
+    <bridge-type id="config">
+        <label>Tankerkoenig Configuration</label>
+        <description>Configuration of Tankerkoenig Bridge, which can handle 1 to 10 gas stations (Tankstellen).</description>
+        <config-description>
+            <parameter name="apikey" type="text" required="true">
+                <label>API-Key</label>
+                <description>Tankerkoenig API-Key. Neccessary registration under https://creativecommons.tankerkoenig.de</description>
+                <default></default>
+            </parameter>
+            <parameter name="refresh" type="integer" required="true" min="10">
+                <label>Refresh Time</label>
+                <description>Sets the refresh time. Minimum is 10 minutes.</description>
+                <default>60</default>
+            </parameter>
+            <parameter name="setupmode" type="boolean">
+                <label>Setup-Mode</label>
+                <description>In mode Setup no data is polled.  After completing the Setup for all stations deactivation this mode will start the polling.</description>
+                <default>true</default>
+            </parameter>
+            <parameter name="use_OpeningTime" type="boolean">
```

Comment:
Will I cooperate all above changes when I'm back home next week. 
Thanks for the inputs.